coalib/misc/Caching.py

```diff
@@ -1,8 +1,8 @@
 import time
 import os
+import logging
```

Comment:
hm the import order was already screwed up... wanna do a new commit sorting those `time` and `os` imports, and then you `import logging` appropriately? :)